.github/workflows/config/release-notes.json

```diff
@@ -56,6 +56,16 @@
         "pattern": "<!--.*-->",
         "flags": "gus",
         "target": ""
+      },
+      {
+        "pattern": "\n\\s*\n",
+        "flags": "gu",
+        "target": "\n"
+      },
+      {
+        "pattern": "\n",
+        "flags": "gu",
+        "target": "\n  "
       }
     ],
```

Comment:
I can't add comments to the JSON, but the first transformer removes blank lines (necessary to make the bulleted list look correct). The second transformer inserts two spaces after every newline (tabbing everything over).